static/external/manyplot.js

```diff
@@ -239,6 +240,25 @@ function createPlotter( canvas, multiFrame, options ) {
         }
     };
 
+    plotter.sortData = function (dataPairs) {
+        dataPairs.sort((a, b) => {
+            // Sanity check that the pairs contain data
+            if (a.yData && a.yData.name && b.yData && b.yData.name) {
```

Comment:
Good thinking.  We've seen some odd cases in practice when hardware malfunctions, misbehaves, etc. so this should save us grief later on.